### DISCUSSION TITLE
Regarding using Anisotropic heat conduction modules

### DISCUSSION THREAD

**Discussion by abhi1587** [idaholab/moose#28608] [Q&A Modules: Thermal Hydraulics] [answered]
Hi, I am trying to simulate a problem with a thick pipe with anisotropic thermal properties and a single fluid core. The issue is the THM module by default adds isotropic heat conduction kernels for the 3d structure. In my case the pipe.

I did solve the issue by editing the source file of 'HeatStructureInterface.C' by adding a flag which when not true, allows me to add the desired anisotropic heat transfer kernels for the pipe. 

I wanted to ask the community if there is another way of achieving the same which does not involve the source code alteration.
Thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

The current workflow is HeatStructureInterface creates a HeatConductionModel which creates the isotropic heat conduction kernels. 
This cannot be changed at the moment.

We are however moving towards having a PhysicsInterface for components, which creates a Physics (see https://mooseframework.inl.gov/syntax/Physics/index.html)
and there you can create your own "heat conduction physics" which would be anisotropic. The existing one is not

An anisotropic option to the existing heat conduction physics would be a welcome contribution

Guillaume
